packages/livechat/src/components/Messages/MessageList/index.js

```diff
@@ -127,6 +131,17 @@ export class MessageList extends MemoizedComponent {
 				continue;
 			}
 
+			const videoConfJoinBlock = message.blocks?.find(({ appId }) => appId === 'videoconf-core')?.elements?.find(({ actionId }) => actionId === 'joinLivechat');
+			if (videoConfJoinBlock) {
+				if (ongoingCall && isCallOngoing(ongoingCall.callStatus)) {
+					const { url, callProvider, rid } = incomingCallAlert || {};
+					items.push(
+						<JoinCallButton callStatus={ongoingCall.callStatus} url={url} callProvider={callProvider} rid={rid} />,
+					);
+				}
+				continue;
+			}
+
```

Comment:
Is this code really required? since LiveChat is totally equipped to deal with UIKit messages so the join call message which you're sending on the core for this video call should be rendered here as well. I tested it locally and here's the result. As you can see, the joincall link is visible to visitor

![image](https://user-images.githubusercontent.com/34130764/173829313-aca66112-68f1-4b87-95f8-71b5708e07a5.png)

Reply:
Btw, this current code seems to be failing due to that translation error

Reply:
I tried not to modify the current behavior, but replacing it with the regular message is fine by me.
Main difference I can see is that the current join button disappears once the call is over and the videoconf message will remain there.

Reply:
since the existing code isn't working due to that weird translation bug, I'd also say let's keep the regular message!!

Reply:
That code with the translation error would still be used by the webrtc option though (I haven't changed anything on how webrtc works)